### PR TITLE
Include 10000 in index description of user guide

### PR DESCRIPTION
**Pull request by Droffilc13**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ For advanced users, the [Command summary](#command-summary) section gives you an
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME e/EMAIL`, `e/EMAIL n/NAME` is also acceptable.
 
-* If a command requires `INDEX` as an input, only one input for `INDEX` is expected. `INDEX` **must** be an integer more than 0 and less than 10,000.<br>
+* If a command requires `INDEX` as an input, only one input for `INDEX` is expected. `INDEX` **must** be an integer from 1 to 10000. <br>
```
Comment (weichang18) [inline docs/UserGuide.md:99]: For 10000 is it better to use 10,000 instead.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ For advanced users, the [Command summary](#command-summary) section gives you an
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME e/EMAIL`, `e/EMAIL n/NAME` is also acceptable.
 
-* If a command requires `INDEX` as an input, only one input for `INDEX` is expected. `INDEX` **must** be an integer from 1 to 10000. <br>
+* If a command requires `INDEX` as an input, only one input for `INDEX` is expected. `INDEX` **must** be an integer from 1 to 10,000. <br>
```
Comment (qinguorui2001) [inline docs/UserGuide.md:99]: not sure if i remembered correctly the index is from 1 to 9999 (less than 10000)

**Review by weichang18 — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 2 above)
Comment (Droffilc13) [reply docs/UserGuide.md:99]: Max is 10000 I believe. I wrote the test cases with it too. Can download our jar file to confirm again

**Pull request merged**